### PR TITLE
fix Linux RID on Travis

### PR DESCRIPTION
**Pull request by anurse**
Going to wait until aspnet/dnx#3048 is merged so I know that it actually works ;). Once this is in, we can remove this from DNX's makefile

/cc @davidfowl @pranavkm

### CODE REVIEW

File: build/_k-standard-goals.shade [hunk 1]
```diff
@@ -40,6 +40,14 @@ default NUGET_FEED = 'https://api.nuget.org/v3/index.json'
   }
 
   Log.Info("Build v2: " + IsBuildV2);
+
+  // It looks like it isn't possible to do this in .travis.yml because we only want
+  // to do it on Linux and I couldn't find an easy way to do that kind of selective
+  // environment variable set.
+  if(E('TRAVIS') == "true" && CanBuildForLinux)
```
Comment (pranavkm) [inline build/_k-standard-goals.shade]: `if (` formatting. Also, does this need to work for Mac - does `CanBuildForLinux` work in that scenario?

**Comment by pranavkm** [comment]
:shipit:

File: build/_k-standard-goals.shade (hunk 1 above)
Comment (anurse) [inline build/_k-standard-goals.shade]: It might... I'll see what the RID story looks like on OSX

**Comment by muratg** [comment]
This good to check-in?

**Comment by davidfowl** [comment]
CanBuildForLinux doesn't exist...

**Comment by anurse** [comment]
This is super old, lets just kill it

**Pull request closed without merging**